src/main/java/oss/fosslight/service/impl/VerificationServiceImpl.java

```diff
@@ -1163,7 +1164,8 @@ public void updateStatusWithConfirm(Project project, OssNotice ossNotice) throws
 		// SPDX
 		String spdxSheetFileId = null;
 		if(CoConstDef.FLAG_YES.equals(project.getAllowDownloadSPDXSheetYn())) {
-			spdxSheetFileId = ExcelDownLoadUtil.getExcelDownloadId("spdx", project.getPrjId(), EXPORT_TEMPLATE_PATH);
+			String dataStr = Maps.immutableEntry("prjId", project.getPrjId()).toString();

```

Comment:
If it is changed to JAVA 11 later, it will be changed to the method below.
```java
String dataStr = Map.of("prjId", project.getPrjId()).toString();
```